src/NzbDrone.Core/Notifications/Webhook/WebhookPayload.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace NzbDrone.Core.Notifications.Webhook
+{
+    internal class WebhookPayload
+    {
+        public string EventType { get; internal set; }
+        public WebhookSeries Series { get; internal set; }
+        public WebhookEpisodeFile EpisodeFile { get; internal set; }
+        public WebhookEpisode Episode { get; internal set; }
```

Comment:
so like:

```json
Episode: {
Airdate: blah,
Airdateutc
EpisodeNumber
SeasonNumber,
Title
},
ReleaseInfo: {
sceneName,
quality,
releaseGroup
}
```

I think sometimes, especially with torrents, season packs can have different scene names and quality?